pkg/metrics/predefined_metrics.go

```diff
@@ -125,6 +126,21 @@ func init() {
 			},
 			tags: []string{group, tagBytes, "drop"},
 		})
+		// DNS metrics
+		dnsLabels := append(labels, "DnsFlagsResponseCode")
+		predefinedMetrics = append(predefinedMetrics, taggedMetricDefinition{
+			PromMetricsItem: flpapi.PromMetricsItem{
+				Name:     fmt.Sprintf("%s_dns_latency_seconds", groupTrimmed),
+				Type:     "histogram",
+				ValueKey: "DnsLatencyMs",
+				Filters: []flpapi.PromMetricsFilter{
+					{Key: "DnsId", Type: flpapi.PromFilterPresence},
+				},
+				Labels:     dnsLabels,
+				ValueScale: 1000, // ms => s
```

Comment:
This scale makes it really difficult to read to me:
![image](https://github.com/netobserv/network-observability-operator/assets/91894519/5160ca31-feaa-4f6c-8979-68567acab67d)

Why don't we show `ms` as same as in plugin ?